sql/core/src/main/scala/org/apache/spark/sql/execution/SparkStrategies.scala

```diff
@@ -307,6 +307,16 @@ private[sql] abstract class SparkStrategies extends QueryPlanner[SparkPlan] {
     }
   }
 
+  object StreamingRelationStrategy extends Strategy {
```

Comment:
Let's add comments to explain that it is just for explain. Also, let's explain that for the actual streaming query, `StreamingRelation` and `StreamingExecutionRelation` will be replaced in `StreamExecution` and `StreamingQueryManager`, respectively. So, this rule will not actually do anything.